website/payments/widgets.py

```diff
@@ -11,10 +11,18 @@ class PaymentWidget(Widget):
     """
 
     template_name = "payments/widgets/payment.html"
+    obj = None
```

Comment:
This creates a class variable instead of an object variable, this line can be removed